score.rb

```diff
@@ -0,0 +1,35 @@
+class Score
+
+  def game_total(frames)
+    add_bonuses(frames)
+    frames.flatten.compact.sum

```

Comment:
This is cool - I've never seen the .compact word!

Reply:
Just read up about it - that is such an elegant way to account for gutter balls (just set those scores to nil, then eliminate them from the score total using compact.)